lib/Backend/LowerMDShared.cpp

```diff
@@ -8683,27 +8684,19 @@ void LowererMD::GenerateFastInlineBuiltInCall(IR::Instr* instr, IR::JnHelperMeth
             //     if (shouldCheckNegZero) {
             //         CMP roundedFloat, 0
             //         JA $setZero
-            //         JE $negZeroTest
-            //         J $bailout
-            //     } else {
-            //         CMP roundedFloat, 0
-            //         JL $bailout
-            //         J $setZero
-            //     }
+            //       $negZeroTest [Helper]:
```

Comment:
it's not technically needed, but our backend isn't happy if you have a path that can only lead to helper inside of the main path. (And this label causes the block to be reordered during layout phase).